### PR TITLE
rule page complete.

### PR DESCRIPTION
**Pull request by sameep-s**
Rules for quiz: page added.
you can see the page below

Screenshot (81)

### CODE REVIEW

**Review by psychoactiv — APPROVED** [review]

**Comment by psychoactiv** [comment]
Code looks good to merge

File: css/rules.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  font-size: 4rem;
+  border-bottom: 0.1rem solid var(--darker);
+}
+
+.rules_points {
+  width: 90%;
+}
+
+.rule-text {
+  font-size: 3rem;
```
Comment (paarthbajaj) [inline css/rules.css:25]: Is your class "rule-text" a generic class? If not you can try making classes like fs-3 or txt-3 and use it whenever you need. Got this advice from the mentor.

File: css/rules.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.main__container {
+  width: 100%;
+  height: 100vh;
+}
+
+.container__rules {
+  width: 80rem;
+  height: 60rem;
+  /* background-color: var(--primary-shade); */
```
Comment (paarthbajaj) [inline css/rules.css]: I guess you do not need this bg color, so why not remove it.

**Review by paarthbajaj — APPROVED** [review]
Looks good. You can merge. Just read my comments once.

File: css/rules.css (hunk 1 above)
Comment (sameep-s) [reply css/rules.css:25]: sure paarth but as of now i have not added font-size utilities in my component library when i will add i'll sure take this approach. thanks.

File: css/rules.css (hunk 2 above)
Comment (sameep-s) [reply css/rules.css]: okay i'll do that.

**Pull request merged**